english/eating-walls/light-fire/fire.md

```diff
@@ -0,0 +1,4 @@
+You build a camp-fire, grab a fistful of the wall and put it on a stick.
```

Comment:
I only want to keep make-some-smores.md as this has no conflicts.  Thank you.
William F. Baur